vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -105,3 +105,238 @@ To add a `Figure` to your page:
 
 If the existing figure functions from [`vizro.figures`](../API-reference/figure-callables.md) do not serve your purpose,
 there is always the option to create a [custom Figure](custom-figures.md).
+
+
+### KPI Cards
+A KPI card is a dynamic card that can display a single value. Optionally, a title, icon and reference value can be added.
+It is a common visual component to display key metrics in a dashboard. Vizro supports two pre-defined KPI card functions:
+
+- [`kpi_card`](../API-reference/figure-callables.md#kpi_card): A KPI card with a single value.
+- [`kpi_card_with_reference`](../API-reference/figure-callables.md#kpi_card_with_reference): A KPI card with a single value and a reference value.
```

Comment:
I see why you've done it, but all the examples on this page have a `title` which is actually an optional argument. And these functions have quite a few different arguments. It's all explained beautifully on the API reference page, but maybe worth reinforcing here also just to be clear. Similarly for the above bullet point.
```suggestion
- [`kpi_card_with_reference`](../API-reference/figure-callables.md#kpi_card_with_reference): A KPI card with a single value and a reference value. Required arguments are `data_frame`, `value_column` and `reference_column`.
```